tests/foreman/api/test_imagemode.py

```diff
@@ -524,3 +526,256 @@ def test_bootc_rex_errata_install(target_sat, bootc_host, function_repos_collect
         must_succeed=True,
         poll_timeout=100,
     )
+
+
+def test_positive_host_packages_persistence_api(target_sat):
+    """Verify that package persistence info is exposed via /api/v2/hosts/:id/packages endpoint
+
+    :id: a1b2c3d4-e5f6-7890-abcd-ef1234567890
+
+    :steps:
+        1. Create a host
+        2. Add packages with different persistence values via SQL (transient, persistent, None)
+        3. Retrieve packages via /api/v2/hosts/:id/packages API endpoint
+        4. Verify persistence info is present and correct for each package
+
+    :expectedresults:
+        1. Packages with persistence='transient' are returned with correct value
+        2. Packages with persistence='persistent' are returned with correct value
+        3. Packages with persistence=None are returned with None/null value
+        4. All packages returned via API include the 'persistence' field
+
+    :Verifies: https://issues.redhat.com/browse/SAT-36788
```

Comment:
Just the issue number
```suggestion
    :Verifies: SAT-36788
```